src/contrib/cluster/Akka.Cluster.Sharding.Tests.MultiNode/ClusterShardingSpec.cs

```diff
@@ -349,32 +433,66 @@ private IActorRef CreateRegion(string typeName, bool rememberEntities)
 
         #region Cluster shardings specs
 
-        [MultiNodeFact(Skip = "TODO")]
+        [MultiNodeFact]
+        public void ClusterSharding_specs()
+        {
+            ClusterSharding_should_setup_shared_journal();
+            ClusterSharding_should_work_in_single_node_cluster();
+            ClusterSharding_should_use_second_node();
+            ClusterSharding_should_support_passivation_and_activation_of_entities();
+            ClusterSharding_should_support_proxy_only_mode();
+            ClusterSharding_should_failover_shards_on_crashed_node();
+            ClusterSharding_should_use_third_and_fourth_node();
+            ClusterSharding_should_recover_coordinator_state_after_coordinator_crash();
+            ClusterSharding_should_rebalance_to_nodes_with_less_shards();
+
+            ClusterSharding_should_be_easy_to_use_with_extensions();
+
+            ClusterSharding_should_be_easy_API_for_starting();
+
+            PersistentClusterShards_should_recover_entities_upon_restart();
+            PersistentClusterShards_should_permanently_stop_entities_which_passivate();
+            PersistentClusterShards_should_restart_entities_which_stop_without_passivation();
+            PersistentClusterShards_should_be_migrated_to_new_regions_upon_region_failure();
+            PersistentClusterShards_should_ensure_rebalance_restarts_shards();
+        }
+
         public void ClusterSharding_should_setup_shared_journal()
         {
             // start the Persistence extension
             Persistence.Persistence.Instance.Apply(Sys);
-            RunOn(() => Sys.ActorOf<MemoryJournal>("store"), _controller);
+            RunOn(() =>
+            {
+                Persistence.Persistence.Instance.Apply(Sys).JournalFor("akka.persistence.journal.MemoryJournal");
+            }, _config.Controller);
             EnterBarrier("persistence-started");
 
             RunOn(() =>
             {
-                Sys.ActorSelection(Node(_controller) / "user" / "store").Tell(new Identify(null));
-                var sharedStore = ExpectMsg<ActorIdentity>().Subject;
-                //SharedLeveldbJournal.setStore(sharedStore, system)
-            }, _first, _second, _third, _fourth, _fifth, _sixth);
+                Sys.ActorSelection(Node(_config.Controller) / "system" / "akka.persistence.journal.MemoryJournal").Tell(new Identify(null));
+                var sharedStore = ExpectMsg<ActorIdentity>(TimeSpan.FromSeconds(10)).Subject;
+                Assert.NotNull(sharedStore);
 
+                MemoryJournalShared.SetStore(sharedStore, Sys);
+            }, _config.First, _config.Second, _config.Third, _config.Fourth, _config.Fifth, _config.Sixth);
             EnterBarrier("after-1");
+
+            RunOn(() =>
+            {
+                //check persistence running
```

Comment:
yes, it's not there. It's checking if the shared journal actor is accessible from other nodes